cogs/moderation.py

```diff
@@ -9,14 +9,18 @@ def __init__(self, bot, config):
         self.bot = bot
         self.config = config
 
+        # compile all patterns into a dictionary as form
```

Comment:
First word of a comment should be capitalized (see [this](https://www.python.org/dev/peps/pep-0008/#comments)).
```suggestion
        # Compile all patterns into a dictionary as form
```